charts/shoot-core/components/charts/apiserver-proxy/values.yaml

```diff
@@ -0,0 +1,11 @@
+
+images:
+  apiserver-proxy: image-repository
+  apiserver-proxy-sidecar: image-repository
+
+advertiseIPAddress: 1.1.1.1
+proxySeedServer: proxy.10.55.61.175.nip.io:8443
```

Comment:
Do you really want to use this value?
The example below is using `dummy.127.0.0.1.nip.io`